add(ttl): Added SQL ttl settings

Kimura-Eitaro opened the pull request:
## background
- Currently, SQL ttl settings are set manually by editing the configmap after building evp-hub.
- Manual settings will require more effort, so you need to change it so that you can build an evp-hub that includes sql ttl settings.

## Fixes
- In the future, we will build evp-hub with values.yaml that describes SQL ttl settings.

maht reviewed (approved):
LGTM. Just to comment to maybe add "timescale" word in the title/description to fit better the context of the change.

Kimura-Eitaro commented:
@maht san
Edited the title/description. Please confirm.
### title
- before
add(ttl): Added ttl settings key
- after
add(ttl): Added SQL ttl settings


### description
- before
```
background
-Currently, Telemetry TTL settings are set manually by editing the configmap after building evp-hub.

Manual settings will require more effort, so you need to change it so that you can build an evp-hub that includes TTL.
Fixes
・In the future, we will build evp-hub with values.yaml that describes Telemetry TTL settings.
```

- after
```
background
- Currently, SQL ttl settings are set manually by editing the configmap after building evp-hub.

Manual settings will require more effort, so you need to change it so that you can build an evp-hub that includes sql ttl settings.
Fixes
- In the future, we will build evp-hub with values.yaml that describes SQL ttl settings.
```
